pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -1,146 +1,91 @@
 { lib
-, stdenv
-, python
-, cmake
-, meson
-, vim
-, ruby
-, which
+
+  # nixpkgs functions
+, buildGoModule
+, buildVimPluginFrom2Nix
 , fetchFromGitHub
-, fetchurl
 , fetchpatch
-, llvmPackages
-, rustPlatform
-, buildGoModule
-, pkg-config
-, curl
-, openssl
-, libgit2
-, libiconv
-, xkb-switch
-, fzf
-, skim
-, stylish-haskell
+, fetchurl
+, stdenv
```

Comment:
stdenv is not a function.

Reply:
Yeah, I wasn't sure about this one. I put it there because it's included to get access to functions like `mkDerivation` and `isDarwin`. I'll move it below `lib` at the top.